### ISSUE TITLE
Title and Description variables don't appear to be passed through correctly.

### ISSUE THREAD

**Issue by iamgrimm** [remg427/TA-thehive-ce#24]
It doesn't appear that the $title$ and $description$ fields are being passed correctly.  The field indicates you can specify a field name that contains the title or description text, but my title was supposed to be "Password hashes dumped from LSASS memory" and instead $title$ passes "notable event".  Instead of the $description$ it is just blank.  The description should have been "LSASS process memory has been read to obtain password hashes, indicating possible credential theft. Attackers can use hashes obtained from LSASS process memory to impersonate or perform actions on behalf of logged on or recently logged on users."  I have no idea where it is getting the field contents that it is passing to theHive.

**Comment by remg427** [comment]
Hello,
in alert action form, you can either provide a string or a field name from results.
For example, you can set a static title in form "Password hashes dumped from LSASS memory" or a dynamic title set in field inline_title. For dynamic title simply put "inline_title" in field tittle without $ before and after. 
It seems that Splunk does not replace token by values: $title$ should be replaced by Splunk at time of calling alert action. 

in other words, alert action code retrieves value from field th_description (description). If empty, alert description is set to "No description provided.". If field th_description  contains a valid field name, then description string is taken from that result field provided it is not null or empty.
